lib/rpmsg/rpmsg_virtio.c

```diff
@@ -24,6 +24,18 @@
 /* Time to wait - In multiple of 1 msecs. */
 #define RPMSG_TICKS_PER_INTERVAL                1000
 
+/* Get the held status */
+#define RPMSG_BUF_HELD_STATUS(rp_hdr)           \
+	(((rp_hdr)->reserved & RPMSG_BUF_HELD_MASK) >> RPMSG_BUF_HELD_SHIFT)
+
+/* Increase held status to keep buffer */
+#define RPMSG_BUF_HELD_INC(rp_hdr)              \
+	((rp_hdr)->reserved += 1 << RPMSG_BUF_HELD_SHIFT)
+
+/* Dncrease held status to keep buffer */
```

Comment:
s/ Dncrease /Decrease

suggestion:
```suggestion
/* Decrease Buffer held counter */
```

Reply:
Done